src/urlBuilder.ts

```diff
@@ -72,21 +72,48 @@ export const parseFilters = (
         if (['like', 'ilike'].includes(operation)) {
             // we split the search term in words
             values = filter[key].trim().split(/\s+/);
+        }
+        // CASE: Logical operator
+        else if (['or', 'and'].includes(operation)) {
+            // we extract each value entries and make it dot separated string.
+            const { filter: subFilter } = parseFilters(
+                { filter: filter[key] },
+                defaultListOp
+            );
+
+            const logics: string[] = [];
```

Comment:
What is logics? To me that is not obvious. Maybe there is another variable name, that is clearer?

Reply:
for me logics is an array that contains the filter expressions, like for example in the test case, `{ "age@lt": 18 }` and `{ "age@gt": 21 }` are two logics or we can say two filters. But since there are many `filter` variables already, then I refer each sub-filters like this as logical operations (hence logic). If it was you, what variable name you think will be more suitable and clearer to represent those stated above?